File: src/main/java/com/eric/Hello.java
package com.eric;

public class Hello {
    public static void main(String[] args) {
        Person p=new Person("Eric",66.5f,1.7f);
        //p.Hello();
       /* p.weight=66.5f;
        p.height=1.7f;
        p.name="Eric";*/
        System.out.println(p.name+p.bmi());
        char c='A';
        System.out.println(c>'a');


    }
}

File: src/main/kotlin/com/eric/Hello.kt
package com.eric

fun main(args: Array<String>) {
//    Human().hello()
    val h = Human(weight =  66.5f, height = 1.7f)
    h.bmi()
    println(h.bmi())
    val c:Char='A'
    println(c.toInt()>60)

  /*  h.hello()
    var age=19
    var name="Eric"
    println(name+age)*/
}
class Human(var weight:Float,var height:Float,var name:String="") {
    init {
        println("test $weight$height")
    }
    fun bmi():Float{
        val bmi=weight/(height*height)
        return bmi

    }
    fun hello() {
        println("Hello kotlin")
    }
}
